PyInstaller/hooks/hook-PyQt5.py

```diff
@@ -21,6 +22,7 @@
 if is_win:
     from PyInstaller.utils.win32.winutils import extend_system_path
     extend_system_path([os.path.join(x, 'PyQt5') for x in getsitepackages()])
+    extend_system_path([os.path.join(os.path.dirname(PyQt5.__file__), 'Qt', 'bin')])
```

Comment:
Please use `get_module_file()`, which is much more save.

When updating a pull-request, you can simply (force) push the updated branch to github again. This will automatically update the pull-request (which follows the branch, not the commit). So you do not need to close the pull-request and open a new one. This also has the benefit that the discussion history is kept.